docs/macvtap-bridge.md

```diff
@@ -0,0 +1,32 @@
+# Using MACVTAP to Bridge onto Host Network
```

Comment:
I think we should clean-up and simplify our `networking.md` document and make MACVTAP the preferred setup. The current one is quite confusing imo.